config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
+
   mount_devise_token_auth_for 'User', at: 'auth'
-  #devise_for :users
+  get 'books', to: 'books#index'
+  get 'books/shows' , to: 'books#shows'
```

Comment:
Let's change this to the `resources` syntax. [Here](https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default) is a general reading about it if you want to take a look.

The REST standard gives us some rules about how resources should be fetched, for example:
- if you want to fetch a collection of books, you should do `GET /books`
- if you want to fetch a single book `GET /books/{id}` (`GET /books/12345`)
- if you want to update a single book `PATCH /books/{id}` including a body with the fields you want to update

So, to be consistent with the standard Rails gives us a helper to create the necessary routes `resources :books`. With this, you will create ALL the routes you need to make "basic" operations on a resource (list all, fetch one, create a new one, update an existing one, delete an existing one).
In this case we only want to list all the books and show a particular one, so we should use it as follow
```ruby
resources :books, only: [:index, :show]
```